### DISCUSSION TITLE
Is the Unifi controller plugin maintained?

### DISCUSSION THREAD

**Discussion by valantur** [mimugmail/opn-repo#123] [Q&A]
Hey I'm new to OPNSense and I was glad I found the Unifi controller as a plugin and a chance to get rid of the cloud key. However, I noticed the version is 8 months old and I believe I read something about the plugin being deprecated soon. Any idea if that's still the case? I don't want to migrate the settings if it turns out the plugin is dying.
Thanks!

**Comment by mimugmail** [comment]
https://www.reddit.com/r/OPNsenseFirewall/comments/we7l7h/unifi7_plugin_finally_available/

**Comment by micceo** [comment]
Awesome work! Got a Q.
Hi, tag along here. Possible to update it so it matches releases from unifi? A saw my other controller (dietpi), was up 7.3.76. Can't figure out how to bring this repos version up to match that. Brgs!

**Reply by micceo** [reply]
Hi again! As general noob on github, is the source available for the plugin unifi7?

**Comment by mimugmail** [comment]
Freebsd always takes a bit longer to add the latest updates. Usually I update them once a month.

**Reply by micceo** [reply]
Had an illusion of taking a peek at the source if it was available. Small chances that I actually was going to understand what I see, but one can always learn new stuff taking a peek into the _unknown_.
Thank you for the swift response, then I can have the controller in OPNsense and not worrying that it's not going to keep being updated.

**Reply by mimugmail** [reply]
You can install the plugin and the source is under /usr/local/opnsense :)
